mw-webapp/src/converter/DayReportConverter.ts

```diff
@@ -9,9 +9,9 @@ import {PlanForNextPeriodService} from "src/service/PlanForNextPeriodService";
 
 const FIRST_INDEX = 0;
 
-const jobsDoneRaw = await JobDoneService.onValueFromRealTimeDb();
-const plansForNextPeriodRaw = await PlanForNextPeriodService.onValueFromRealTimeDb();
-const problemsForCurrentPeriodRaw = await CurrentProblemService.onValueFromRealTimeDb();
+const jobsDoneRaw = await JobDoneService.getJobsDone();
+const plansForNextPeriodRaw = await PlanForNextPeriodService.getPlansForNextPeriod();
+const problemsForCurrentPeriodRaw = await CurrentProblemService.getCurrentProblems();
```

Comment:
why do you have async data in the top level of module
why do you have get data logic inside converter. Converter should just convert data from one format to another

Reply:
Delete async data with logic from converter